### DISCUSSION TITLE
Store object content as part of the xl.meta file.

### DISCUSSION THREAD

**Discussion by mirisu2** [minio/minio#16411] [Q&A] [answered]
Hi,

from [this](https://blog.min.io/minio-optimizes-small-objects/) topic:
`Beginning with [MinIO’s 2021-04-22 release](https://github.com/minio/minio/releases), the server can store object content as part of the xl.meta file that stores per object metadata. A few factors determine when this is done, but generally files less than 128KiB are likely to be stored inline with the metadata. This reduces the IOPS needed both to read and write the files.`

Is this behavior configurable? I mean, can I, to configure the size of object to be as part of the xl.meta file?
For example, I would like to store the object as part of the xl.meta file for files which size more then 250k?

**Comment by klauspost** [comment, marked as answer]
@mirisu2 No. This is not configurable. There are too many factors that play into this for you to make a reasonable decision on it.

**Comment by wuchencm** [comment]
What is the default buffer size in the latest version of MinIO?

**Reply by harshavardhana** [reply]
https://github.com/minio/minio/blob/master/cmd/erasure-object.go#L1402
